addons/crm_iap_lead_enrich/models/res_config_settings.py

```diff
@@ -1,21 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import api, models
-
-
-class ResConfigSettings(models.TransientModel):
-    _inherit = "res.config.settings"
-
-    @api.model
-    def get_values(self):
-        values = super(ResConfigSettings, self).get_values()
-        cron = self.sudo().with_context(active_test=False).env.ref('crm_iap_lead_enrich.ir_cron_lead_enrichment', raise_if_not_found=False)
-        values['lead_enrich_auto'] = 'auto' if cron and cron.active else 'manual'
-        return values
-
-    def set_values(self):
-        super(ResConfigSettings, self).set_values()
-        cron = self.sudo().with_context(active_test=False).env.ref('crm_iap_lead_enrich.ir_cron_lead_enrichment', raise_if_not_found=False)
-        if cron:
-            cron.active = self.lead_enrich_auto == 'auto'
```

Comment:
Again, not sure why the setting and the cron are de-synchronized now.

Reply:
Done